content/faqs.md

```diff
@@ -38,9 +38,8 @@ Newspeak involved multiple changes but the most essential were:
 
 Everything in the conscious language initiative is *the exact opposite* of that.
 
-For example, "Denylist" is both more precise and more accurate than "blacklist";
-
-"Primary" is both more precise and more accurate than "master" in projects that have made the switch.
+For example, "Primary" is both more precise and more accurate than "master" in projects that have
+made the switch.
```

Comment:
I would suggest that we consider also documenting Secondary in this context, as there are other uses of 'master' being replaced (like master -> main in github repos) that involve the same replacement with quite different replacement word case.  It feels like also documenting secondary here would disambiguate.

Reply:
That's fine by me. I'll add that now.